backend/views/syson-diagram-statetransition-view/src/main/java/org/eclipse/syson/diagram/statetransition/view/STVDescriptionNameGenerator.java

```diff
@@ -21,7 +24,34 @@
  */
 public class STVDescriptionNameGenerator extends DescriptionNameGenerator {
 
+    private static final String ACTION = "action";
+
+    private static final String ACTIONS = "actions";
+
+    private static final String SPACE = " ";
+
+    private final String stvDiagramPrefix;
```

Comment:
Could you instead modify `DescriptionNameGenerator` by adding an accessor method for the `diagramPrefix` attribute or by changing the visibility of `diagramPrefix`?